src/components/HeaderComponent/HeaderComponent.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import LoginIcon from '../../assets/icons/log_out.svg?react';
```

Comment:
creatre alise for import of assets
for example 
`@assets/icons/logout.svg`
https://dev.to/navdeepm20/spoiler-alert-import-alias-in-react-vite-3apf

Reply:
Fixed